cmd/describe_stacks_test.go

```diff
@@ -64,6 +77,18 @@ func TestSetFlagValueInDescribeStacksCliArgs(t *testing.T) {
 		},
 		{
 			name: "No flags changed, set default format",
+			initFlags: func(fs *pflag.FlagSet) {
+				fs.String("file", "", "Write the result to file")
+				fs.String("format", "yaml", "Specify the output format (`yaml` is default)")
+				fs.StringP("stack", "s", "", "Filter by a specific stack\nThe filter supports names of the top-level stack manifests (including subfolder paths), and `atmos` stack names (derived from the context vars)")
+				fs.StringSlice("components", nil, "Filter by specific `atmos` components")
```

Comment:
These test cases would not have caught the bug that we faced with issue: https://github.com/cloudposse/atmos/issues/1309

It is testing `setCliArgsForDescribeStackCli` if that is what we are looking for.